docs/cluster-deployment/stripe.rst

```diff
@@ -75,6 +75,17 @@ panel.
     The storage capacity of a cluster cannot currently be changed in the
     CrateDB Cloud Console once it has been deployed.
 
+Shared-resources clusters (CR0)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Shared-resource clusters are a special category of clusters that allow
+better utilisation of resources. These cluster essentially share resources with
```

Comment:
I'm not 100% sure of the precise technicalities but maybe something like this reads a bit more clearly

```suggestion
These clusters share compute and storage resources with other clusters in this category.
```